CMakeLists.txt

```diff
@@ -7,6 +7,15 @@ set(CppUTest_version_minor 8)
 # 3.1 is needed for target_sources
 cmake_minimum_required(VERSION 3.1)
 
+###############
+# Conan support
+###############
+if (EXISTS "${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
+    include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+    message(STATUS "conan_basic_setups()")
```

Comment:
Is there a reason that "conan_basic_setups()" is plural here?

Reply:
Sorry it's a typo error. I will fix this tomorow.

Reply:
I'll merge on new commit to avoid the need for a new PR

Reply:
@basvodde, the typo is fixed.